registry/data-models/models.py

```diff
@@ -0,0 +1,171 @@
+from pydantic import BaseModel
+from typing import List
+
+"""
+This file defines abstract backend data models for feature registry.
+Backend data models will be used by backend API server to talk to feature registry backend.
+Purpose of this is to decouple backend data models from API specific data models.
+For each feature registry provider/implementation, they will extend this abstract
+data models and backend API.
+Diagram of the data models:
```

Comment:
Just one suggestion, possible we use mermaid for this flowchart? [Mermaid is officially supported in github](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/). This will make furhter maintain easiler.

Reply:
Thanks for the suggestion, I added diagram using mermaid.